### PR TITLE
**Feature:** Add initial Debug Overlay component

### PR DESCRIPTION
**Pull request by TejasQ**
# Summary
This component adds functionality that would make a floating debug viewer possible inside all of our applications that takes a simple map of values to display to the user.

According to the CIP, it is drag-and-droppable as well.

## Todos
- [x] Add copy-to-clipboard that copies relevant info about the current page.
- [x] Add a test for [`makeRowsFromObject`](https://github.com/contiamo/operational-ui/pull/740/files#diff-611f8412fa66fee3f4ab27365cde5330R62).

# To be tested

## Me
- [x] No error or warning in the console on `localhost:6060`
image

## You

- [x] Things look good on the demo.
- [x] I can move around the viewer in order to get a better view.
- [x] I can copy and paste meaningful data by clicking the copy button.

### CODE REVIEW

File: src/Debug/Debug.tsx [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  className?: string
+  expanded?: boolean
+}
+
+export interface DebugState {
+  isExpanded: boolean
+  top?: number
+  left?: number
+
+  // Required for calcuating final drop position.
```
Comment (fabien0102) [inline src/Debug/Debug.tsx]: typo ;)

File: src/Debug/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+A component that displays information intended to convey meaningful information about the current running environment to the end-user.
+
+## Usage
+
+```jsx
+const styled = require("react-emotion")
```
Comment (fabien0102) [inline src/Debug/README.md]: You can import styled from index logically

File: src/Debug/Debug.tsx (hunk 1 above)
Comment (TejasQ) [reply src/Debug/Debug.tsx]: UFFFF

File: src/Debug/README.md (hunk 2 above)
Comment (TejasQ) [reply src/Debug/README.md]: ok

**Comment by TejasQ** [comment]
[Debug viewer](https://trello.com/c/vriPdTua/419-debug-viewer)

**Pull request merged**